documentation/docs/deploy/installation.md

```diff
@@ -13,6 +13,10 @@ Integration Server based on Gradle and docker images. Therefore, you have to hav
 * Docker Compose  
 * Gradle 6+
 
+If you want to provision a Kubernetes cluster on cloud with help of terraform first you have to ensure that you have installed it along with the
```

Comment:
Still not clear what CLI tol we are talking about, and what version has to be there. Requirements are vague.